src/Analyzers/CSharp/CodeFixes/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -85,30 +85,7 @@ internal static class ConvertToRecordEngine
         var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
         // first see if we need to re-order our primary constructor parameters.
```

Comment:
note: this code needs a *lot* of cleanup work.  Fix is targeted to the definite assumption problem causing the crash.  but a whoel rewrite here would be appropriate.